test/runTests.cpp

```diff
@@ -586,11 +615,39 @@ bool runTests() {
     return allTestsPassed;
 }
 
+
+/**
+ * Tests if AlphabetTooBig exception is thrown when the alphabet size is too big
+ */
+bool testException(){
+    // Here we generate two same sequences. They are having distinct element for
+    // each position, so the length of each sequence is same its alphabet size.
+    // An alphabet size of 260 should make edlibAlign throw AlphabetTooBig exception,
+    // since it is larger than 256, the largest value that can be saved in an unsigned character.
+    int seqLength = 260;
+    char* query = static_cast<char *>(malloc(sizeof(char) * seqLength));
+    char* target = static_cast<char *>(malloc(sizeof(char) * seqLength));
+    bool pass = false;
+    for (int i = 0; i < seqLength; i++) {
+        query[i] = static_cast<char>(i);
+        target[i] = static_cast<char>(i);
+    }
+    try {
+        EdlibAlignResult result =
+            edlibAlign(query, seqLength, target, seqLength, edlibDefaultAlignConfig());
+        edlibFreeAlignResult(result);
+    }catch(const AlphabetTooBigException& e) {
+        pass = true;
```

Comment:
You could skip having `pass` variable and instead do:
```
catch (const AlphabetTooBigException &e) {
    printf("\x1B[32m""OK""\x1B[0m\n");
    return true;
}
printf("\x1B[31m""FAIL""\x1B[0m\n");
return false;
```

But, if you like the `pass` solution better, that is also ok!